requirements/sharepoint-on-premises-user-permissions-requirements.md

```diff
@@ -56,11 +56,11 @@ These privileges will be granted automatically when a new SPDocKit database is c
 
 ### Workstation Requirements
 
-When running the application on a workstation, SPDocKit user connecting to any SharePoint 2010, SharePoint 2013, SharePoint 2016 or SharePoint Online site will need to be granted __Site Collection Administrator__ privileges for each site he is trying to explore and manage.
+When running the application on a workstation, SPDocKit user connecting to any SharePoint 2010, SharePoint 2013 or SharePoint 2016 site will need to be granted __Site Collection Administrator__ privileges for each site he is trying to explore and manage.
 
 To connect to an existing SPDocKit database from the workstation, SPDocKit user will need to be granted __db_datareader__ rights on SPDocKit database.
```

Comment:
To connect to an existing SPDocKit database from the workstation, __the__ SPDocKit user will need to be granted ___the__ __db_datareader__ rights on SPDocKit database.